diff --git a/python/MANIFEST.in b/python/MANIFEST.in
index 9ee720972..171829908 100644
--- a/python/MANIFEST.in
+++ b/python/MANIFEST.in
@@ -1 +1 @@
-include ray/python/ray/autoscaler/ray-schema.json
\ No newline at end of file
+include ray/autoscaler/ray-schema.json
diff --git a/python/setup.py b/python/setup.py
index 64623177b..d2c98070d 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -298,7 +298,8 @@ def build(build_python, build_java):
     bazel_targets += ["//java:ray_java_pkg"] if build_java else []
     return bazel_invoke(
         subprocess.check_call,
-        ["build", "--verbose_failures", "--"] + bazel_targets,
+        ["build", "--verbose_failures", "--jobs=40", "--action_env=CONDACC=$CC",
+         "--"] + bazel_targets,
         env=bazel_env)

